### DISCUSSION TITLE
[REQUEST] Multiple Price

### DISCUSSION THREAD

**Discussion by karno68** [Blair2004/NexoPOS#476] [Ideas]
suggestions or input for adding more features than one price in the product :
when I use the din-in category, the price is sell1, but when I use the delivery category the price will change to price 2, 

can you make sell1 - sell10 features in one product?
one product has many prices according to the sales category of sales

**Comment by Blair2004** [comment]
Hi, 
Thank you for your ideas, we don't plan to add such features on the system for the meantime.

**Reply by boudo2017** [reply]
sell price should be in-line editable then you can do that.

**Reply by Blair2004** [reply]
The sale price can be edited on the cart. From the settings it should be enabled. 

Settings > POS > Features
